app/views/declaration/additionalInformtion/additional_information_required.scala.html

```diff
@@ -48,7 +49,7 @@
     backButton = Some(BackButton(messages("site.back"), Navigator.backLink(AdditionalInformationRequired, mode, ItemId(itemId))))) {
 
     @formHelper(action = controllers.declaration.routes.AdditionalInformationRequiredController.submitForm(mode, (itemId)), 'autoComplete -> "off") {
-        @errorSummary(form.errors)
+        @errorSummary(radioGroupErrors("yesNo", "required_Yes", form.errors))
```

Comment:
Yes.  There are quite a few places where we could now use yesNoErrors instead of all the different variations.    I might look at doing that and see if I can get the PR even bigger.